### PR TITLE
Issue #2872, Entering multiple email addresses in a Contact form cate…

### PR DESCRIPTION
**Pull request by opi**
…gory can be confusing.

### CODE REVIEW

**Comment by opi** [comment]
oups, dirty git history... will need some cleanup at some point :/

File: core/modules/contact/contact.admin.inc [hunk 1]
```diff
@@ -258,13 +259,18 @@ function contact_category_edit_form_validate($form, &$form_state) {
     }
   }
 
+  // Get addresses separated by comma, a space or line break.
```
Comment (opi) [inline core/modules/contact/contact.admin.inc]: incomplete comment (missing semicolon info).

File: core/modules/contact/contact.admin.inc [hunk 2]
```diff
@@ -194,9 +194,13 @@ function contact_category_edit_form($form, &$form_state, array $category = array
...
     '#title' => t('Recipients'),
-    '#default_value' => $category['recipients'],
-    '#description' => t("Example: 'webmaster@example.com' or 'sales@example.com,support@example.com' . To specify multiple recipients, separate each e-mail address with a comma."),
+    // Recipients are stored comma-separated, but display with new lines for
+    // better human readability.
+    '#default_value' => implode("\n", explode(',', $category['recipients'])),
+    '#description' => t("To specify multiple recipients, separate each e-mail address with a new line, comma, semicolon or space."),
+    '#placeholder' => t("webmaster@example.com\nsales@example.com"),
     '#required' => TRUE,
+    '#rows' => max(5, sizeof(explode(',', $category['recipients'])) + 2),
```
Comment (klonos) [inline core/modules/contact/contact.admin.inc:203]: Clever 👍

**Comment by quicksketch** [comment]
Thanks, merged into 1.x for 1.11.0!

**Pull request merged**